spec/features/pages/dashboard.rb

```diff
@@ -30,6 +30,19 @@ class Dashboard < SitePrism::Page
     element :search_field, '.search_module input#forms_search_prison_number'
     element :search_button, '.search_module input.search_button'
     element :search_escorts_due_button, '.search-header button.go'
+    element :create_new_escort, 'input[type="submit"][value="Start new PER"]'
```

Comment:
This stuff has been moved to the search page, it's not on the dashboard page anymore:

spec/features/pages/search.rb